modules/local/xml_vcf/resources/usr/bin/shortvariant.py

```diff
@@ -174,11 +188,26 @@ def main():
     # Create VCF headers
     vcf_headers = create_vcf_header(date_str, chrs, chr_dic, input_file_name)
 
+    # Define condition for indels (more than one character in REF or ALT)
+    indel_condition = (df_processed['REF'].str.len() > 1) | (df_processed['ALT'].str.len() > 1)
```

Comment:
I noticed that there is a xml field `variant-type` in `short_variants` defined in the XML files to distinguish SNV and InDel.